packages/garmin-connect/src/adapters/http/garmin-http-client.ts

```
@@ -0,0 +1,177 @@
+    const cons = await fetchConsumer();
+    oauth2Token = await exchangeOAuth2(oauth1Token, cons, fetchFn);
+    logger.info("OAuth2 token refreshed");
+  };
+
+  const waitForRefresh = (): Promise<string> =>
+    new Promise((resolve, reject) => {
+      refreshSubscribers.push({ resolve, reject });
+    });
+
+  const notifySubscribers = (): void => {
+    refreshSubscribers.forEach((s) => s.resolve(oauth2Token!.access_token));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)